src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1411,28 +1411,7 @@ internal void SaveAs(string path)
             }
         }
 
-        /// <summary>
-        /// Save the specified workspace to a file. If the file path is null or empty, an
-        /// exception is written to the console.
-        /// </summary>
-        /// <param name="path">The path to the file.</param>
-        /// <param name="workspace">The Workspace to save.</param>
-        internal void SaveAs(string path, WorkspaceModel workspace)
```

Comment:
I think this method is not required.

Reply:
The other `SaveAs` only saves `currentWorkspace` but seems this one aims to save any workspace, can you check Ian's original PR and why added this one before removal?

Reply:
This saveAs saves only currentWorkspaceModel and the other SaveAs saves both CurrentSpaceViewModel and currentWorkspaceModel.  Here is the commit which added this  saveAs : https://github.com/DynamoDS/Dynamo/commit/8b4db7f58d0b4f3be5b4e317e77df7297d21e975
My guess is this was before viewBlock serialization. Might be wrong.